### PR TITLE
Added ScreenCordinates 

### PR DESCRIPTION
**Pull request by Rloaded**
I added ScreenCoordinates to Atlas so that we can change between Screen- and GeoCoordinates(LatLng). Therefore, I added a new class ScreenCoordinate and created two new abstract methods getLatLng() and getScreenCoordinate().

### CODE REVIEW

File: packages/atlas/lib/src/screen_coordinate.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class ScreenCoordinate {
+  /// The x coordinate in screen pixels.
+  final int x;
+
+  /// The y coordinate in screen pixels.
+  final int y;
+
+  const ScreenCoordinate({
+    @required this.x,
+    @required this.y,
```
Comment (Rloaded) [inline packages/atlas/lib/src/screen_coordinate.dart:15]: I know that in other classes the constructor has a slightly different approach ("@required int x, f.e). For me this ended in a compiler error, so I went with that approach. Feel free to let me know, how I could change it so that it works with your CodingGuidelines.

File: example/ios/Flutter/flutter_export_environment.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (dakrawczyk) [inline example/ios/Flutter/flutter_export_environment.sh]: This file should not be included in the commit.

File: example/.flutter-plugins-dependencies [hunk 3]
```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"google_maps_flutter","dependencies":["flutter_plugin_android_lifecycle"]}]}
```
Comment (dakrawczyk) [inline example/.flutter-plugins-dependencies]: This file should not be included in the commit.

File: packages/google_atlas/example/.flutter-plugins-dependencies [hunk 4]
```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"google_maps_flutter","dependencies":["flutter_plugin_android_lifecycle"]}]}
```
Comment (dakrawczyk) [inline packages/google_atlas/example/.flutter-plugins-dependencies]: This file should not be included in the commit

File: packages/atlas/lib/src/screen_coordinate.dart [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/foundation.dart';
+
+/// A pair of x and y coordinates.
+/// The `x` and `y` coordinates are stored as screen pixels (not display pixels)
+/// relative to the top of the map, not top left of the whole screen.
+class ScreenCoordinate {
```
Comment (dakrawczyk) [inline packages/atlas/lib/src/screen_coordinate.dart]: I feel like this class should be called `ScreenCoordinates` instead of `ScreenCoordinate`
@kmjaris @noahmateen do you agree?

File: packages/atlas/lib/src/screen_coordinate.dart [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+/// The `x` and `y` coordinates are stored as screen pixels (not display pixels)
+/// relative to the top of the map, not top left of the whole screen.
+class ScreenCoordinate {
+  /// The x coordinate in screen pixels.
+  final int x;
+
+  /// The y coordinate in screen pixels.
+  final int y;
+
+  const ScreenCoordinate({
```
Comment (felangel) [inline packages/atlas/lib/src/screen_coordinate.dart]: we should override `==` and `hashCode`

File: packages/atlas/test/screen_coordinate_test.dart [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import 'package:atlas/atlas.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+main() {
```
Comment (felangel) [inline packages/atlas/test/screen_coordinate_test.dart]:
```suggestion
void main() {
```

**Review by felangel — APPROVED** [review]

**Review by felangel — CHANGES_REQUESTED** [review]
see comments

**Review by felangel — DISMISSED** [review]

**Review by noahmateen — APPROVED** [review]

**Review by dakrawczyk — APPROVED** [review]
